kratos/input_output/logger_output.cpp

```diff
@@ -43,6 +43,9 @@ namespace Kratos
         auto message_severity = TheMessage.GetSeverity();
         if (TheMessage.WriteInThisRank() && message_severity <= mSeverity)
         {
+            if (message_severity == LoggerMessage::Severity::WARNING)
+                mrStream << "[WARNING] ";
```

Comment:
Please also check for `TheMessage.IsDistributed()` (probably merge with the if below?), otherwise in MPI you will get as many tags as processes.

Reply:
Actually, disregard this comment, this is already only printed once for MPI :man_facepalming: